Python/flowgraph.c

```diff
@@ -1558,6 +1558,41 @@ optimize_if_const_op(basicblock *bb, int n, PyObject *consts, PyObject *const_ca
     return ERROR;
 }
 
+static int
+optimize_if_const_list_or_set_iter(basicblock *bb, int n,
```

Comment:
```suggestion
optimize_if_const_list_or_set_iter(basicblock *bb, int i,
```

I found it confusing to use `n` for the last index rather than the number of item.